### PR TITLE
[Feature] Add disclaimer to account deletion text

### PR DESCRIPTION
**Pull request by rediris**
This paltry legalese is to remind users to download their certificates when deleting their account data.

See https://github.com/gymnasium/tracker/issues/185

### CODE REVIEW

**Review by andrewpmiller — COMMENTED** [review]
Consider:
Be advised that deleting your data will remove any Accredible certificates or badges that you have earned on the Gymnasium. Please back up your certificates or badges immediately if you plan on keeping them.

**Comment by rediris** [comment]
@andrewpmiller sounds good!

**Comment by jgagne** [comment]
> Please back up your certificates or badges immediately if you plan on keeping them.

Also consider:
Please back up (or download) your certificates or badges immediately if you plan on keeping them.

Too wordy?

**Review by jgagne — DISMISSED** [review]
